tools/build_versions.py

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env python
 
 from pathlib import Path
-import requirements
 import pkg_resources
+import re
 
 
 def main():
     requirements_dir = Path(__file__).parent / '..' / 'requirements'
 
     for requirement_file in requirements_dir.glob('*.txt'):
         print(requirement_file.name)
-        with open(str(requirement_file), 'r') as fd:
-            for req in requirements.parse(fd):
+        with open(str(requirement_file), 'r') as f:
+            for req in f:
+                req = req.strip()
+                if req[0] == '#':
+                    continue
+                req = re.split('<|>|=|!|;', req)[0]
                 try:
-                    version = pkg_resources.get_distribution(req.name).version
-                    print(req.name.rjust(20), version)
+                    version = pkg_resources.get_distribution(req).version
+                    print(req.rjust(20), version)
```

Comment:
Does this code not care at all about the version limiters that are in the file? The new code doesn't, but it looks like the old code didn't either? Is that what we want?

Reply:
Oh I see, this is just printing what's installed. Sorry, I didn't read this PR carefully and thought all this logic was part of the setup.py. I'm coming around to your point of view. =)